Tests/Integration/DotNet/Azure/CosmosDb/CosmosDbStorageTests.cs

```diff
@@ -0,0 +1,131 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder;
+using Xunit;
+
+namespace IntegrationTests.Azure.CosmosDb
+{
+    [Trait("TestCategory", "Deprecated")]
+    [Trait("TestCategory", "Storage")]
+    [Trait("TestCategory", "CosmosDb")]
```

Comment:
We can leave the Deprecated category as it could be useful in the future (v5 release), but the other two should be removed.